### PR TITLE
Add command to toggle dark mode

### PR DESCRIPTION
**Pull request by unnamedd**
# Toggle dark mode
toggle-dark-mode

# How it works
Open Raycast and type "toggle system appearance".

# Result
The system will alternate between the light mode and the dark mode.

### CODE REVIEW

File: system/toggle-dark-mode.applescript [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Dark Mode
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🦉
```
Comment (PitNikola) [inline system/toggle-dark-mode.applescript:9]: Suggestion for emoji to reflect switching between dark/light: 🌗

File: system/toggle-dark-mode.applescript [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Dark Mode
```
Comment (PitNikola) [inline system/toggle-dark-mode.applescript:5]: How about naming it "Toggle System Appearance" since it's not only about the dark mode?

**Review by PitNikola — APPROVED** [review]
Nice one! We actually wanted to add it as built in command, but wanted to sort out permission request dialog in a nicer way first and it got de-prioritized. Let's have it as a script anyway, might be useful to some folks. I have only minor suggestion to the naming and emoji :)

File: system/toggle-dark-mode.applescript (hunk 1 above)
Comment (unnamedd) [reply system/toggle-dark-mode.applescript]: I thought about this icon and I decided to use the owl anyway, but for sure, this one is better! Thanks for the suggestion! :D

File: system/toggle-dark-mode.applescript (hunk 2 above)
Comment (unnamedd) [reply system/toggle-dark-mode.applescript]: Done!

**Pull request merged**